Microsoft.Research/ManagedContract.Setup/DotNet.Contracts.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <CodeContractsInstallDir>$(MSBuildThisFileDirectory)</CodeContractsInstallDir>
```

Comment:
Why would you want to override the value for "where is Code Contracts" to something other than "right here" when using the NuGet package?